src/megaclient.cpp

```diff
@@ -9826,7 +9869,9 @@ error MegaClient::openfilelink(const char* link, int op)
     {
         ptr += 8;
 
-        if (*ptr == '!')
+        int a = strlen(ptr);
+        int b = sizeof key;
```

Comment:
Are the above variables for testing leftovers? :)

Reply:
Fixed it